waspc/e2e-test/GoldenTest.hs

```diff
@@ -77,6 +77,7 @@ runGoldenTest goldenTest = do
 
     isTestOutputFileTestable :: FilePath -> Bool
```

Comment:
```suggestion
    shouldTestOutputFileBeCompared :: FilePath -> Bool
```
not sure how to go about this one, but I was just a bit confused at first because we use the word "test" in two different contexts - first one (isTestOutputFile) refers to running the specific version of the compiler with specific input, and then the 2nd one (Testable) refers to the comparison of two such inputs (golden and current, as we call them).

That is why I tried to somehow delineate those two, some ideas: `shouldTestOutputFileBeIncludedInComparison` (huh :D), `shouldTestOutputFileBeCompared`. I also used `should` instead of `is` because that made it sound less like "testability" is the inherent property of a file, but more like something that we decided from "outside" for some reason. But I don't think "is" is wrong, this is just what I had in my ear when thinking about it.

But not a big deal altogether, just wanted to share my thoughts here in case they are helpful. I am aware you all already had a discussion about the naming here so please feel free to proceed with whatever you are the most comfortable with.